### ISSUE TITLE
What exactly does `/textual_file` support?

### ISSUE THREAD

**Issue by aretrace** [KoStard/HermesCLI#25]
What file formats does `/textual_file` support? Any non-binary file?

**Comment by KoStard** [comment]
Yes, PDF, Word, PPTX, XLSX, and more. If you test something else, please share, will include in the docs
